services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -153,6 +154,8 @@ class DynamicSidecarSettings(BaseCustomSettings):
         description="Names the traefik zone for services that must be accessible from platform http entrypoint",
     )
 
+    R_CLONE_SETTINGS: RCloneSettings = Field(auto_default_from_env=True)
```

Comment:
All fields in a settings class must have the same prefix (those that don't, is because they were not refactored)